app/src/main/java/com/example/musictoned/Navigation.kt

```diff
@@ -150,21 +153,24 @@ fun MusicTonedNavHost(navController: NavHostController = rememberNavController()
             )
         }
 
-        composable(EDIT_ROUTINE_ROUTE) {
+        composable("$EDIT_ROUTINE_ROUTE/{exerciseName}") {
```

Comment:
Should this be `{routineName}`?

Reply:
This specifically passes back a single Exercise object name to be added to the current edited routine so I think this naming is okay.